src/SortableList.js

```diff
@@ -87,8 +89,7 @@ export default class SortableList extends Component {
   componentWillReceiveProps(nextProps) {
     const {data, order} = this.state;
     let {data: nextData, order: nextOrder} = nextProps;
-
-    if (data && nextData && !shallowEqual(data, nextData)) {
```

Comment:
@gitim, the check for `shallowEqual` here will always return false when the data is a nested object.
Should https://github.com/gitim/react-native-sortable-list/blob/v0.0.16/src/utils/shallowEqual.js#L17
instead be: 
- ignoring object comparisons,
- or doing a deep comparison, via `JSON.stringify(objA[keysA[i]]) !== JSON.stringify(objB[keysA[i]])`?

Thanks!

Reply:
Do you mean if the data is part of a nested object? (i.e. redux?).

Reply:
For my case, data is a map of keys to values, where some values are maps themselves. When comparing the maps against each other, `shallowEqual` will always return false.

e.g.
```
a = {1: "a", 2: "b", 3: "c"};
b = {1: "a", 2: "b", 3: "c"};

myData = {:key1 a};    // shallowEqual will always return false with this data

a === b;                                 // returns false
JSON.stringify(a) === JSON.stringify(b); // returns true
```

Reply:
Interesting.. I think I'm also using nested values (each row represents a model with varied maps and attributes) but I'm not having any issues getting the new orders.

Reply:
It was my initial intension, because it is not expected that nested objects will be cloned